src/app/public/modules/avatar/avatar.inner.component.html

```diff
@@ -1,12 +1,30 @@
-<div class="sky-avatar-wrapper">
-  <div class="sky-avatar-image" [hidden]="!src"></div>
+<div
+  class="sky-elevation-1 sky-avatar-wrapper"
+  [ngClass]="'sky-avatar-wrapper-size-' + (size || 'large')"
+>
+  <div
+    class="sky-avatar-image"
+    [hidden]="!src"
+  ></div>
   <div
     class="sky-avatar-initials"
-    [ngClass]="'sky-palette-multi-' + (colorIndex + 1)"
     [hidden]="src || !name"
+    [ngClass]="'sky-palette-multi-' + (colorIndex + 1)"
+  >
+    <div
+      class="sky-avatar-initials-inner"
+      [ngClass]="
+        (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'modern' ?
```

Comment:
Should we move this logic into the typescript file so it'll show up in coverage?

Reply:
I intentionally kept this in the template as it deals with CSS classes, and because the `async` pipe will automatically clean up subscriptions when the component is destroyed. I did write tests for it, as well as visual regression tests that ensure the styles are applied regardless of the class names, so I think we're well covered.